ng2-components/ng2-activiti-processlist/src/services/activiti-process.service.ts

```diff
@@ -43,6 +43,19 @@ export class ActivitiProcessService {
             .catch(err => this.handleError(err));
     }
 
+    /**
+     * Retrieve deployed apps details by id
+     * @param appId - number - optional - The id of app
+     * @returns {Observable<any>}
+     */
+    getApplicationDetailsById(appId: number): Observable<any> {
+        return Observable.fromPromise(this.apiService.getInstance().activiti.appsApi.getAppDefinitions())
+            .map((response: any) => {
+                return response.data.find(p => p.id === appId);
+            })
```

Comment:
Please we don't use one variable name, can you give a name that give the idea of the parameter thanks